generator/lib/src/entity_resolver.dart

```diff
@@ -125,6 +125,9 @@ class EntityResolver extends Builder {
       _propertyChecker.runIfMatches(f, (annotation) {
         propUid = annotation.getField('uid')!.toIntValue();
         fieldType = propertyTypeFromAnnotation(annotation.getField('type')!);
+        if (!annotation.getField('signed')!.toBoolValue()!) {
+          flags |= OBXPropertyFlags.UNSIGNED;
+        }
```

Comment:
Does it make sense to add the flag for any type? E.g. should this error if the property is used on a not supported type?

Reply:
Letting core to deal with that at the moment.

Reply:
e.g. error-out during model creation